### ISSUE TITLE
How to predict my custom data?

### ISSUE THREAD

**Issue by Qinstudy** [sharathadavanne/seld-dcase2019#7]
@sharathadavanne @bharatsubedi I have also been experimenting with custom datasets recently. I wonder if it is possible to use the 35 audio provided by bharatsubedi as an evaluation data set and then observe the prediction results.
In other words,using the data set provided by sharathadavanne as a training set, my custom data set is used as a test set, and then the azimuth and elevation of the sound event are predicted. Can it be feasible?

**Comment by sharathadavanne** [comment]
@Qinstudy As long as the training and hardware conditions are similar between the provided dataset and @bharatsubedi dataset, I dont see why it should not work.

**Comment by Qinstudy** [comment]
Hi, @sharathadavanne  I want to use your Ambisonic dataset as a training set, and then I will make a test dataset  similar to your dataset. 
The scheme is as follows: my current hardware condition is 16 microphone arrays,  playing the 11 sound categories provided by DCASE2016 in the front of these 16  microphone arrays, and moving the sound source slowly, then creating wav audio files and csv metadata, and make 50 audio files, each for 60 seconds. 

Is this scheme feasible? In  a word, using your Ambisonic dataset as a training set, and then I collect data and make my own test set. Can this scenario predict the desired result? Thank you for your generous help.

**Comment by sharathadavanne** [comment]
Hi @Qinstudy since the training dataset consists of four channel audio in the Ambisonic format, the test dataset should also be of four channel Ambisonic format. If you wish to use your 16 mic array as the testing dataset then you will have to train the network with the same 16 mic array training dataset.

**Comment by Qinstudy** [comment]
thank you for your advice, I will have a try!
